src/_actions/assetMakes.actions.js

```diff
@@ -33,8 +33,10 @@ export const loadAssetMakes = (pageNumber, limit) => (dispatch) => {
     );
 };
 
-export const loadAssetMakesDropdown = () => dispatch =>
-  axios.get('asset-makes/?paginate=false')
+export const loadAssetMakesDropdown = () => (dispatch) => {
+  dispatch({ type: LOADING_ASSET_MAKES });
```

Comment:
Make this its own action. That is:
```
const loadingAssets = loading => ({
  type: LOADING_ASSET_MAKES,
  loading
})
```
This way, you don't have to "piggy-back" the success/error actions for setting the loading state.
Please see an [example usage here](https://github.com/sachungo/movies/blob/master/resources/assets/js/actions/index.js#L11) and [here](https://github.com/sachungo/movies/blob/master/resources/assets/js/reducers/moviesReducer.js#L12)